tests/lsdb_rubin/test_tract_patch_search.py

```diff
@@ -0,0 +1,102 @@
+from lsdb.catalog import Catalog
+from lsdb_rubin.tract_patch_search import tract_patch_search
+
+Catalog.tract_patch_search = tract_patch_search
```

Comment:
What.

This is terrifying.

I mean, I guess I see what it's doing, and it's probably terribly pythonic.

Reply:
Right? I wasn't too sure about the best way to integrate this kind of add-on package to be honest, so if you/anyone (now or later) has a preferred solution, I'm all ears :)